Alpaca.Markets/Interfaces/IPortfolioHistory.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Alpaca.Markets
+{
+    /// <summary>
+    /// Encapsulates portfolio history information from Alpaca REST API.
+    /// </summary>
+    public interface IPortfolioHistory
+    {
+        /// <summary>
+        /// Gets historical equity values.
+        /// </summary>
+        IReadOnlyList<Decimal?>? Equity { get; }
+
+        /// <summary>
+        /// Gets historical profit/loss values.
+        /// </summary>
+        IReadOnlyList<Decimal?> ProfitLoss { get; }
+
+        /// <summary>
+        /// Gets historical profit/loss values as percentages.
+        /// </summary>
+        IReadOnlyList<Decimal?> ProfitLossPct { get; }
```

Comment:
Yes, values can be null. The way it works is that there's going to be a timestamp for each period (timestamps can't be null.) But if for whatever reason there's no value for that time period in any of the other lists, there will be null values there. Filtering the null values would prevent matching up the other lists to the timestamp list (by index value.)

Reply:
OK: Got it now after your explanation and reviewing the docs. But I think in this case it would be better to provide users with combined list of items instead of 4 separate lists. Would you mind if I'll implement these changes in the PR branch?